vmngclient/tests/templates/models/cisco_aaa.py

```diff
@@ -0,0 +1,44 @@
+# type: ignore
```

Comment:
Can we be more specific what type-check error we are ignoring?
Is it really needed for whole file?
(found also in other files)